app/tests/spec/views/settings/gravatar_permissions.js

```diff
@@ -0,0 +1,121 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'jquery',
+  'sinon',
+  'views/settings/gravatar_permissions',
+  'lib/metrics',
+  'models/user',
+  '../../../mocks/router',
+  '../../../lib/helpers'
+],
+function (chai, $, sinon, View, Metrics, User, RouterMock, TestHelpers) {
+  'use strict';
+
+  var assert = chai.assert;
+
+  describe('views/gravatar_permissions', function () {
+    var view;
+    var routerMock;
+    var metrics;
+    var user;
+    var email;
+    var account;
+    var SERVICE_NAME = 'Gravatar';
+    var CLIENT_ID = 'gravatar';
+    var PERMISSIONS = ['profile:email'];
```

Comment:
Can these constants be made into static constants on the View to avoid duplication & possible breakages if the values are updated?